Added: Express Checkout (#165)
vippsas/vipps-epayment-api: faq.md

# Frequently asked questions

> For common questions, see: [Common API FAQ](https://developer.vippsmobilepay.com/docs/vipps-developers/faqs)

Reviewer: ```suggestion
For common questions, see [Common API FAQ](https://developer.vippsmobilepay.com/docs/vipps-developers/faqs).
```

## Is Express Checkout available in the ePayment API?

Not yet.

> It's possible to impement the ePayment API for everything except Express Checkout,

Reviewer: ```suggestion
It's possible to implement the ePayment API for everything except Express Checkout,
```

and use the
[eCom API](https://developer.vippsmobilepay.com/docs/APIs/ecom-api/)
just for
[Express Checkout](https://developer.vippsmobilepay.com/docs/APIs/ecom-api/vipps-ecom-api/#express-checkout-payments).

As mentioned on the [API platform overview page](https://developer.vippsmobilepay.com/docs/APIs):

> "With the Vipps MobilePay API platform, you benefit from a shared API framework for all the APIs. This means that all APIs use the same API keys, authentication methods, terminology, and error message formats. Integrating with our APIs is straightforward, and combining functionalities from multiple APIs is easy."

Reviewer: Adding in the exact quote from another page is risky because the text changes often and will quickly be out of date. It's better to rephrase it, if you need it to be here, and add the link for more information (if there is more information there).

Author: Yup, but I think we need this, and the link is already there..

> "With the Vipps MobilePay API platform, you benefit from a shared API framework for all the APIs. This means that all APIs use the same API keys, authentication methods, terminology, and error message formats. Integrating with our APIs is straightforward, and combining functionalities from multiple APIs is easy."

Author: ```suggestion
With the Vipps MobilePay API platform, you benefit from a shared API framework for all the APIs. This means that all APIs use the same API keys, authentication methods, terminology, and error message formats. Integrating with our APIs is straightforward, and combining functionalities from multiple APIs is easy.
```

## Can payments be "mixed and matched" between the eCom API and the ePayment API?

> Yes, to a certain extent: If the payments were initiated with the

Reviewer: ```suggestion
Yes, to a certain extent. If the payments were initiated with the
```

> Yes, to a certain extent: If the payments were initiated with the

Reviewer: ```suggestion
Yes, to a certain extent. If the payments are initiated with the
```

[eCom API](https://developer.vippsmobilepay.com/docs/APIs/ecom-api/)

> it's possible to  captured, cancel, etc. using the ePayment API.

Reviewer: ```suggestion
it's possible to capture, cancel, query, and adjust using the ePayment API.
```

Reviewer: It's best not to use "etc." because it looks like we couldn't be bothered to finish the sentence.

> The eCom API uses the ePayment API “behind the scenes”.

Reviewer: ```suggestion
The eCom API uses the ePayment API *behind the scenes*.
```
